package.json

```diff
@@ -65,6 +66,7 @@
   },
   "dependencies": {
     "adm-zip": "^0.4.9",
+    "later": "^1.2.0",
```

Comment:
In the project's [readme](https://github.com/bunkat/later#this-project-is-no-longer-being-maintained-it-should-be-treated-as-sample-code-on-one-technique-to-calculate-future-instances-of-complex-schedules-if-you-are-looking-to-fire-a-task-every-1-minute-use-a-cron-scheduler-instead):
> This project is no longer being maintained.

Might be an issue ;)

Reply:
Indeed it is 🤔 now to find an alternative